extensions/geode-modules-test/src/main/java/org/apache/geode/modules/session/catalina/DeltaSessionJUnitTest.java

```diff
@@ -195,6 +199,65 @@ public void serializeReturnsSerializedObject() throws IOException {
     assertThat(result).isEqualTo(serializedObj);
   }
 
+  @Test
+  public void setDeserializedAttributesValueInvokesGetAttribute() {
+    when(manager.getPreferDeserializedForm()).thenReturn(true);
+    DeltaSession session = spy(new DeltaSession(manager));
+    String attributeName = "name";
+    List<String> attributeList = new ArrayList<>();
+    attributeList.add(attributeName);
+    Enumeration<String> attributeNames = Collections.enumeration(attributeList);
+    doReturn(attributeNames).when(session).getAttributeNames();
+    doReturn(new Object()).when(session).getAttribute(attributeName, false);
+
+    session.setDeserializedAttributesValue();
+
+    verify(session).getAttribute(attributeName, false);
+  }
+
+  @Test
+  public void setDeserializedAttributesValueNotInvokeGetAttributeIfNotPreferDeserializedForm() {
+    when(manager.getPreferDeserializedForm()).thenReturn(false);
+    DeltaSession session = spy(new DeltaSession(manager));
+    String attributeName = "name";
```

Comment:
This string is already defined as a field in the class, so may not need to be defined again here.